store/tikv/2pc.go

```diff
@@ -811,14 +811,30 @@ func (c *twoPhaseCommitter) stripNoNeedCommitKeys() {
 	c.mutations = m.subRange(0, newIdx)
 }
 
+// SchemaVer is the infoSchema which will return the schema version
```

Comment:
Please add `.` at the end of a complete sentence.

There are many such problems in this PR, please modify them together.